sdk/mobile-center-updates/src/test/java/com/microsoft/azure/mobile/updates/UpdatesBeforeDownloadTest.java

```diff
@@ -64,14 +64,13 @@ public ServiceCall answer(InvocationOnMock invocation) throws Throwable {
         ReleaseDetails releaseDetails = mock(ReleaseDetails.class);
         when(releaseDetails.getId()).thenReturn("someId");
         when(ReleaseDetails.parse(anyString())).thenReturn(releaseDetails);
-        Context context = mock(Context.class);
-        when(context.getPackageName()).thenReturn("com.contoso");
-        PackageManager packageManager = mock(PackageManager.class);
-        when(context.getPackageManager()).thenReturn(packageManager);
-        when(packageManager.getPackageInfo("com.contoso", 0)).thenThrow(new PackageManager.NameNotFoundException());
+
+        /* Override mock answer by calling it once then configuring new answer. */
+        mPackageManager.getPackageInfo("com.contoso", 0);
```

Comment:
I thought it will replace the mock even though it hasn't been called. Hmm. I should be careful when I need to do it.